Lib/test/test_typing.py

```diff
@@ -4001,6 +4001,7 @@ def cleanup(self):
 
     @classmethod
     def setUpClass(cls):
+        raise ValueError
```

Comment:
This is intentional. Let's see the CI output, it should be red.

Reply:
It failed as intended: https://github.com/python/cpython/runs/4943128810?check_suite_focus=true#step:5:1483 🎉